spring-integration-cassandra/src/test/java/org/springframework/integration/cassandra/config/outbound-adapter-parser-config.xml

```diff
@@ -0,0 +1,87 @@
+	xsi:schemaLocation="http://www.springframework.org/schema/integration http://www.springframework.org/schema/integration/spring-integration.xsd
+		http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+		http://www.springframework.org/schema/data/cassandra http://www.springframework.org/schema/data/cassandra/spring-cassandra.xsd
+		http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context.xsd
+		http://www.springframework.org/schema/integration/cassandra http://www.springframework.org/schema/integration/cassandra/spring-integration-cassandra.xsd
+		http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util.xsd">
+
+
+	<context:property-placeholder location="classpath:cassandra.properties" />
+	
+	<int:poller default="true" fixed-delay="50" />
+
+	<cassandra:mapping entity-base-packages="org.springframework.integration.cassandra.test.domain">
+		<cassandra:entity
+			class="org.springframework.integration.cassandra.test.domain.Book">
+			<cassandra:table name="book" />
+		</cassandra:entity>
+	</cassandra:mapping>
+
+	<cassandra:cluster contact-points="${cassandra.contactpoints}" port="${cassandra.port}">
+		<cassandra:keyspace action="CREATE_DROP" name="${cassandra.keyspace}" />
+	</cassandra:cluster>
+
+	<cassandra:session keyspace-name="${cassandra.keyspace}" schema-action="RECREATE"/>
+
+
+	
+	<cassandra:converter />
+	<cassandra:template id="cassandraTemplate"  />
+
+	<int-cassandra:outbound-channel-adapter id="cassandraMessageHandler1" 
+							cassandra-template="cassandraTemplate" 
+							type="INSERT"
+							auto-startup="true" />
+							
+	<bean id="writeOptions" class="org.springframework.cassandra.core.WriteOptions">
+		<property name="ttl" value="60" />
+		<property name="consistencyLevel">
+			<util:constant
+				static-field="org.springframework.cassandra.core.ConsistencyLevel.ONE" />
+		</property>
+		<property name="retryPolicy">
+			<util:constant
+				static-field="org.springframework.cassandra.core.RetryPolicy.DOWNGRADING_CONSISTENCY" />
+		</property>
+	</bean>
+	
+	<bean id="resultChannel" class="org.springframework.integration.channel.QueueChannel"/>
+	
+							
+     <int-cassandra:outbound-channel-adapter id="cassandraMessageHandler2" 
+							cassandra-template="cassandraTemplate" 
+							write-options="writeOptions"
+							auto-startup="true" />
+
+	<int-cassandra:outbound-channel-adapter id="cassandraMessageHandler3" 
+				  	cassandra-template="cassandraTemplate"
+					cql-ingest="insert into book (isbn, title, author, pages, saleDate, isInStock) values (?, ?, ?, ?, ?, ?)"
+					auto-startup="true" />
+
+	<int-cassandra:outbound-channel-adapter id="cassandraMessageHandler4" 
+							cassandra-template="cassandraTemplate" 
+							statement-expression="T(com.datastax.driver.core.querybuilder.QueryBuilder).truncate('book')"
```

Comment:
Minor, but see comment in the `CassandraMessageHandler`:
```
/*
 * Register the Cassandra Query DSL package so they don't need a FQCN for QueryBuilder, for example.
 */
((StandardTypeLocator) typeLocator).registerImport("com.datastax.driver.core.querybuilder");
```
So, you can omit the package in this expression.